src/fractional_coloring.jl

```diff
@@ -0,0 +1,56 @@
+function indvec(s, n)
+    x = zeros(n)
+    x[s] .= 1
+    return x
+end
+
+"""
+    fractional_chromatic_number(g; optimizer)
+
+Compute the fractional chromatic number of a graph.  Gives the same result as
```

Comment:
Can you add a source defining the fractional chromatic number? At first I thought it was the solution of the linear relaxation of the coloring ILP

Reply:
It would also be nice to add the description of the problem to `docs/src/algorithms.md`

Reply:
It probably can be viewed as a linear relaxation of an ILP, though there are perhaps multiple ways of doing that.  I've added references to Wolfram MathWorld.